diff --git a/dist/index.js b/dist/index.js
index b40511d151f0ca3a9127042c274c6539083e4b77..55e7f31a436468f661fedbf7b0b6c929ef19fcbb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1290,7 +1290,7 @@ function getReadableNameForPcbTrace(soup, pcb_trace_id) {
 var getReadableNameForElement = (soup, elm) => {
   if (typeof elm === "string") {
     const elmObj = getElementById(soup, elm);
-    if (!elmObj) `unknown (could not find element with id ${elm})`;
+    if (!elmObj) return `unknown (could not find element with id ${elm})`;
     return getReadableNameForElement(soup, elmObj);
   }
   switch (elm.type) {
